docs/src/examples/components/Dropdown/Types/DropdownExampleSingleSearch.shorthand.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Dropdown } from '@stardust-ui/react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownExample = () => (
+  <Dropdown
+    search
+    items={inputItems}
+    placeholder="Start typing a name"
+    getA11ySelectionMessage={{
+      onAdd: item => `${item} has been selected.`,
+    }}
```

Comment:
The same thing about `getA11ySelectionMessage`

Reply:
no, removed, thx